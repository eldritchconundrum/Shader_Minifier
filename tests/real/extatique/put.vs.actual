/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef PUT_VS_ACTUAL_
# define PUT_VS_ACTUAL_

const char *put_vs =
 "void main()"
 "{"
   "gl_FrontColor=gl_Color,gl_Position=ftransform();"
 "}";

#endif // PUT_VS_ACTUAL_
