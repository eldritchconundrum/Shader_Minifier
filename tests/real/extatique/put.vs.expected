/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef PUT_VS_EXPECTED_
# define PUT_VS_EXPECTED_

const char *put_vs =
 "void main()"
 "{"
   "gl_FrontColor=gl_Color,gl_Position=ftransform();"
 "}";

#endif // PUT_VS_EXPECTED_
